pkg/components/linkerd/manifest.go

```diff
@@ -27,4 +36,59 @@ identity:
 {{ .Cert }}
       keyPEM: |
 {{ .Key }}
+
+# Following is from values-ha.yaml which contains the values needed to enable HA mode.
+enablePodAntiAffinity: true
+
+# controller configuration
+controllerReplicas: 3
+controllerResources: &controller_resources
+  cpu: &controller_resources_cpu
+    limit: "1"
+    request: 100m
+  memory:
+    limit: 250Mi
+    request: 50Mi
+destinationResources: *controller_resources
+publicAPIResources: *controller_resources
+
+# identity configuration
+identityResources:
+  cpu: *controller_resources_cpu
+  memory:
+    limit: 250Mi
+    request: 10Mi
+
+# grafana configuration
```

Comment:
Why do we need separate Grafana?

Reply:
Yes it is tightly integrated with the linkerd dashboard

![Screenshot from 2020-07-07 15-01-08](https://user-images.githubusercontent.com/5815795/86760474-df114380-c062-11ea-991a-882a78c6da84.png)